### PR TITLE
(PUP-5064) Block Windows Server 2003 Install

### PR DESCRIPTION
**Pull request by ferventcoder**
Windows Server 2003 is no longer supported.

windows2003

### CODE REVIEW

**Comment by ferventcoder** [comment]
image

File: wix/puppet.wxs [hunk 1]
```diff
@@ -57,9 +57,13 @@
       <!-- http://wixtoolset.org/documentation/manual/v3/howtos/redistributables_and_install_checks/block_install_on_os.html
       http://windows-installer-xml-wix-toolset.687559.n2.nabble.com/How-to-add-newline-in-lt-Condition-message-td705721.html
```
Comment (haus) [inline wix/puppet.wxs]: I think this comment can go away now that newlines aren't required in the condition message. The first link in the comment is still totally handy though.

**Comment by haus** [comment]
I can't say how much I love this.

File: wix/puppet.wxs (hunk 1 above)
Comment (ferventcoder) [inline wix/puppet.wxs]: Good point.

**Comment by ferventcoder** [comment]
@MFredette @nfagerlund the language good in the message above? "Puppet is no longer supported on Windows Server 2003."

File: wix/puppet.wxs (hunk 1 above)
Comment (ferventcoder) [inline wix/puppet.wxs]: Fixed

**Comment by haus** [comment]
:+1: from me

**Comment by MFredette** [comment]
Thumbs up from me as well. The only thing I'd add is something like, "You must upgrade to Windows 2008 (or whatever) to use Puppet." But that is somewhat implied so it's not mandatory.

**Comment by ferventcoder** [comment]
Sweet. @haus @joshcooper @Iristyle @jpogran This is ready to go. Merge away. :)

**Comment by nfagerlund** [comment]
Yup :+1: here too.

**Comment by Iristyle** [comment]
I believe we need to remove some additional assets from this repo that were introduced for the warning, as they're no longer necessary.

Please see https://github.com/puppetlabs/puppet_for_the_win/pull/116 and https://github.com/puppetlabs/puppet_for_the_win/commit/312ef72373f08dfb60c02c81638f18a9f63e47cb

**Comment by ferventcoder** [comment]
Agreed. Maint issue or followup for PUP-5064?

**Comment by Iristyle** [comment]
@ferventcoder might as well tag it `PUP-5064` as well I think...

**Comment by ferventcoder** [comment]
okay

**Comment by ferventcoder** [comment]
https://github.com/puppetlabs/puppet_for_the_win/pull/130 - should I adjust the commit now?

**Pull request merged**